### ISSUE TITLE
Ss infantry able to attack enemy Fighters

### ISSUE THREAD

**Issue by LaughtersGrave** [nicupavel/openpanzer#163]
So you can see the enemy fighter profile here

![screenshot_2018-11-09-01-00-33-154_com android chrome](https://user-images.githubusercontent.com/44041995/48236668-360cd400-e3bb-11e8-9626-ca2de617aeeb.png)

And this is one of the three SS infantry you start the campaign with, it has aggressive attack and tenacious defence from its leader, and good experience... But... I don't think it should be able to attack the fighter... Yet, it seems to be able to, see picture below:

![screenshot_2018-11-09-01-00-13-156_com android chrome](https://user-images.githubusercontent.com/44041995/48236736-81bf7d80-e3bb-11e8-9448-8cb47125e978.png)

**Comment by LaughtersGrave** [comment]
To make this weirder the tiger can attack it too??

I don't get what's happening.

**Comment by LaughtersGrave** [comment]
![screenshot_2018-11-09-01-30-15-488_com android chrome](https://user-images.githubusercontent.com/44041995/48237619-5c347300-e3bf-11e8-82d8-a146a1af0173.png)

**Comment by barsoom1717** [comment]
Another issue I have related to this is enemy anti aircaft being káble to move to get within their orange, and then attack aircraft.  In real life this would be impossible.  Either in PG II or OpenGeneral they corrected this.

**Comment by LaughtersGrave** [comment]
? Anti aircraft can move then fire it is mobile. If it is deployed then some still can. The Bofours 40mm takes a literal 20 seconds to deploy.

This is a turn based game, events do not happen concurrently.

If you have a separate issue start a thread. 

This is about a glitch with units that shouldn't be able to shoot aircraft, not game mechanics.

**Comment by salexgit** [comment]
Hi all

Game mechanics allow ground units to attack air units, which is controlled by a special unit attribute. In principle, infantry or a tank equipped with heavy machine guns can fight back, as it was happing e.g. for Soviets who did not have enough efficient anti-air guns.
Of course, not every infantry should be able to do that and I cannot recall Tiger to have an external machine gun.

**Comment by LaughtersGrave** [comment]
Ok well if this is normal game play this can be closed.

It is not something I'd experienced in earlier game play with the same ss units

**Comment by salexgit** [comment]
Hi

The only discrepancy I can see is that some tanks, e.g. German Tiger, can attack air units. I am not an expert on German warfare but I do not think it can do that.

**Comment by LaughtersGrave** [comment]
Standard tiger 1 cupola MG was limited to 15 degrees of elevation, this is the same on most German vehicles until late war when the panther had a proper MG cupola that allowed it to swing to either side freely and elevate almost 90° and they were far from common.

Don't get me wrong, everything could technically attack air units, but almost nothing armoured could elevate to anything useful to fight aircraft and no infantry used weapons realistically were stable enough to hit aircraft. 

There was a commanders anti aircraft machine gun in the tiger 1 and they were not costly or limited production or anything but they were almost never used by German commanders, who felt they were better employed fighting the tank and the MG was far too ineffective anyway. 

Panther V Ausf G with AA Mount
![screenshot_2018-11-10-19-24-17-958_com android chrome 01](https://user-images.githubusercontent.com/44041995/48305219-b5ff7f00-e51e-11e8-9229-e4c75a55f287.png)

**Comment by salexgit** [comment]
Hi

Fully agree... It would be more logical to disallow tanks to attack air units.
 
The issue is anyway quite marginal because the tank and infantry air attack strength is quite small, just 1. At least I never use these units to attack airplanes.

**Comment by LaughtersGrave** [comment]
Agree. I was just surprised and wondered if it was meant to be allowed.

My I infantry is pretty hardened, with 1200 experience I was able to shoot down 3 planes :D it was luck of course, the result was predicted to be 1 - 2

Given that it was and is very weak. I will close the thread.

**Comment by salexgit** [comment]
Hi LaughtersGrave

While reading some military books, I came across some interesting facts that are relevant to this discussion. It seems that some late German tanks, such as Pz VI Tiger II and Pz IIIL, have a special turret "Fliegerbeschussgerät 42" for the MG42 gun that was installed as an AA weapon. In that sense it is maybe Ok that a tank can attack air targets. 

Yet on the other hand, almost all the sources mention that it is was used merely for the defensive purpose, so it can be viewed as just increased defensive factor against aircrafts, not as attacking one. Just my thoughts... 

Regards, Alex
